common/fuzzing/carbon.proto

```diff
@@ -173,9 +177,10 @@ message Expression {
     TypeTypeLiteral type_type_literal = 19;
     UnimplementedExpression unimplemented_expression = 20;
     ArrayTypeLiteral array_type_literal = 21;
-    CompoundMemberAccessExpression compound_member_access = 22;
-    WhereExpression where = 23;
-    DesignatorExpression designator = 24;
+    ImplicitSizedArrayTypeLiteral implicit_sized_array_type_literal = 22;
+    CompoundMemberAccessExpression compound_member_access = 23;
+    WhereExpression where = 24;
+    DesignatorExpression designator = 25;
```

Comment:
It's generally not a good practice to renumber protocol message fields, because it breaks the serialization format. I'm not sure if we're actually using the serialization format, but it seems better to play it safe, so I'd recommend giving the new field number 25. It can still be next to `array_type_literal`, because they don't have to be in numeric order.